### DISCUSSION TITLE
Facciamo il punto della situazione

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#8] [General]
Immagino abbiate ascoltato la puntata (ahem… _episodio_) [di ieri](https://www.ilpost.it/episodes/ep-48-quello-con-le-console-con-il-deserto-intorno/) e quindi sapete che abbiamo la stima degli autori. Zampa mi ha anche scritto che per la redazione de Il Post non ci sono problemi a procedere.

Le idee in campo in questo momento sono 2:

1. da @bionicco: motore di ricerca di titoli di giochi, magari con annotazioni del tipo "recensione" o "consiglio", chi ne ha parlato e quando (episodio e minutaggio);
2. da @Mascalian: un analogo motore di ricerca per frasi memorabili. Anche questo è molto interessante ma per ora è un'idea un po' vaga, nel senso che… cosa dovrebbe cercare l'utente? "Asse Y"? Ed esce l'episodio 48? C'è da approfondire, probabilmente per le frasi memorabili sarebbe meglio avere non tanto una ricerca quanto un elenco di episodi con l'elenco delle frasi da ricordare, chi l'ha detta ed il minutaggio.

**In ogni caso, per entrambe le idee, serve il testo completo di tutti gli episodi.**

Siccome ritengo davvero impraticabile riascoltare tutto da capo ho iniziato a "giocare" con il servizio di AWS [Amazon Transcribe](https://aws.amazon.com/it/transcribe/) che fa _speech recognition_. Dico subito che il risultato è… buono ma tutt'altro che perfetto. Il costo è abbordabile, per tutti gli episodi dovremmo stare sotto i 100 €.

Ci sono altri servizi di trascrizione in cui è proprio un essere umano a trascrivere il testo: sono ovviamente molto più accurati ma… costano tantissimo e quindi eviterei, parliamo di cifre da **70 a 90 centesimi di € al minuto** di conversazione.

Per affrontare questi costi (eventuali) ho aderito al programma di _sponsorship_ di GitHub, che trovate [qui](https://github.com/sponsors/pdonadeo): in questo modo è possibile donare qualche Euro per affrontare le spese, **se riteniamo di procedere su questa strada**.

Ovviamente c'è chi, vinto dall'entusiasmo compulsivo, ha già donato 10 $, vero @agofa? 😉 

Quindi il progetto ha già 10 $ da spendere ma aspettiamo di capire se l'idea piace a tutti o no.

Apro un altro thread più tecnico (sicuramente interessa a @bionicco) con i risultati della "passata" dell'episodio di ieri attraverso Amazon Transcribe.

Tenete d'occhio il [forum di questo progetto](https://github.com/pdonadeo/e-uscito-joypad/discussions).

**Comment by pdonadeo** [comment]
Trovate i primi risultati con AWS nella [nuova discussione](https://github.com/pdonadeo/e-uscito-joypad/discussions/9).

**Comment by bionicco** [comment]
Io o dei dubbi sulla necessità di trascrivere le puntate.
Sicuramente è utile, ma da solo non basta, e se comunque c'è da fare del lavoro "umano" per analizzarle, tanto vale farlo direttamente sull'audio.
Con il solo testo è possibile are una ricerca full text, ma per le frasi famose potrebbe portare a una quantità di risultati enorme (a meno che uno non si ricordi la frase esattamente e non sia stata trascritta tutta giusta) e anche per i giochi, cercando solo in base al fatto che il titolo sia stato pronunciato, diversi risultati se non classificati potrebbero essere poco significativi. ( Inoltre potrebbe esser carino vedere per ogni puntata quali sono i suoi contenuti)
Quindi se dobbiamo fare le trascrizioni e poi anche un'analisi (sul testo o sull'audio) non so se è un gran vantaggio (dato che comunque non è una cosa a costo zero, sia per tempo che per soldi)

**Reply by pdonadeo** [reply]
Sono d'accordo ma vedila in quest'ottica: ho fatto delle prove per esplorare la possibilità di avere il testo dei podcast, non sono necessariamente convinto di andare fino in fondo con le trascrizioni automatiche. La trascrizione, non perfetta, è possibile ed ora sappiamo questa cosa.

Però tieni presente che anche se dovessimo riascoltare tutto (molto probabile) avere già la trascrizione non è come dover trascrivere a mano. L'episodio 48 sono (solo testo, senza HTML) 39 KiB di testo... che nessuno deve riscrivere ma al massimo leggere mentre ascolta.

Inoltre: nel mio HTML non è evidenziato il minutaggio ma tieni presente che ogni segmento di testo (sono i `<p>` nel mio HTML) è corredato di istante di inizio e di fine. Forse questa è l'informazione più preziosa, pensandoci bene.

**Reply by agofa** [reply]
Concordo con @pdonadeo .
Nel fine settimana (prima gnafò) proverò a riascoltare l'episod ... la punt ... l'event ...insomma il "coso" 48 seguendo la trascrizione e segnando il minutaggio delle citazioni di giochi o autori o produttori
Vedremo...

**Reply by pdonadeo** [reply]
@agofa non serve, ho messo anche il minutaggio nell'output: [postprocess.html.txt](https://github.com/pdonadeo/e-uscito-joypad/files/8609219/postprocess.html.txt)

_(file da rinominare in .html perché GitHub non fa allegare gli html)_

**Reply by agofa** [reply]
Mi sembrava che nel precedente html non ci fosse, meglio!

**Reply by pdonadeo** [reply]
Non c'era, l'ho aggiunto ora. Nell'output di AWS ci sono i secondi per ogni segmento di testo.

**Comment by Mascalian** [comment]
Ciao,
    la trascrizione è sempre utile, tanto più se poi ci si deve fare del
lavoro sopra, e soprattutto per la questione del minutaggio automatico.

Detto ciò, in generale il lavoro sugli episodi passati secondo me è da fare
con calma, all'indietro e un po' alla volta.
Invece, sui nuovi episodi si potrebbe cominciare subito.
Se poi i tre riusciranno a fare podcast più frequenti (o regolari), nel
giro di poco ci troveremo già con un po' di dati.

Riguardo la mia stramba idea delle frasi celebri, intendevo soprattutto
collegare le puntate a momenti particolarmente cominci (o intensi): il
minutaggio non è necessario, anzi; mentre la citazione stringata potrebbe
diventare una sorta di tag o, se ripetuta in altri episodi, di meme.
Così come si potrebbero raccogliere in rubriche ironiche tipo: "le voci di
Bordone", "Zampa-time", "le licenze toscane" (immagino che abbiate capito a
cosa mi stia riferendo).
E qui spiego il perché di questa idea: Joypad lo ascolto raramente per
documentarmi (seguo già Everyeye.it, leggo Manettini con voracità - i
caposaldi - e tanto altro in giro), ma soprattutto per i siparietti, la
leggerezza e quella sensazione di giustezza che mi danno i nostri tre.

mk
